ui/pages/login.tsx

```diff
@@ -366,8 +366,7 @@ const Login: NextPage = () => {
         <>
           {isWebauthn && isSequencedLogin && (
             <p className="u-text--muted">
-              Another layer of authentication before you get access{" "}
-              {getTitleSuffix()}
+              Additional authentication needed to get access {getTitleSuffix()}
```

Comment:
nitpick: Could you change the wording according to the designs?
```suggestion
              Additional authentication needed to access {getTitleSuffix()}
```

Reply:
I wanted to reuse the same function for the title in here. That's why I slightly adjusted the copy so it fits. I think it should be fine though. Both copies seem equivalent to me. Wdyt?

Reply:
The underlying problem this is aiming to solve is that getTitleSuffix can return "to foo" and "foo" depending on other circumstances. Hence the idea to find a copy here that is fit for both.